tests/test_taking_a_vacation.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+import pytest
+
+from codecademy.taking_a_vacation import hotel_cost
+from codecademy.taking_a_vacation import plane_ride_cost
+from codecademy.taking_a_vacation import rental_car_cost
+from codecademy.taking_a_vacation import trip_cost
+
+@pytest.mark.parametrize("test_hotel_cost_input,test_hotel_cost_expected", [
+    (1, 140),
+    (2, 2*140),
+    (7, 7*140),
+    (31, 31*140),
+    (365, 365*140)
+])
+def test_hotel_cost(test_hotel_cost_input, test_hotel_cost_expected):
+    assert hotel_cost(test_hotel_cost_input) == test_hotel_cost_expected
+
+
+@pytest.mark.parametrize("test_hotel_cost_negative_input", [1, -1, "blah", False, None])
+def test_hotel_cost_negative(test_hotel_cost_negative_input):
+    try:
+        hotel_cost(test_hotel_cost_negative_input)
```

Comment:
@09041985 i totally missed this, sorry
this test expects an exception, but if there is no exception thrown your test will do nothing about that.
basically if `hotel_cost()` returns normally you don't have a mechanism to fail the test.
This situation is called that your test may `falsely pass` which is a very bad practice!
Here if your call to `hotel_cost()` passes then Python interpreter will look for next line in the `try` block - you don't have any lines there so it will "jump out" of the try-except block and look for next line  - you don't have any >> test is over and since you didn't throw any exceptions test will pass.
To fix this you need to fail on the situation when `hotel_cost()` doesn't through exception - so just add a line `assert False, "Expected an exception from hotel_cost"` or after hotel_cost() call.
or better save result of calling hotel_cost and add that result to your custom message to `assert False` so it'll be easier to debug failed test.
Please apply this fix to all tests that expect exceptions.